docs/examples/basic_meter/calling_conventions.py

```diff
@@ -53,11 +45,18 @@
     unit="1",
     value_type=int,
     metric_type=Counter,
-    label_keys=("environment",),
 )
 
 labels = {"environment": "staging"}
 
+# Views are used to define an aggregation type to use for a specific metric
+counter_view = View(requests_counter, CountAggregation())
+clicks_view = View(clicks_counter, CountAggregation())
```

Comment:
It looks like the creating a view requires a reference to the metric instrument itself. This means we can create a view only after creating the instrument itself.

Can we instead create Views with the name/description of the instrument? This can also allow us to load Views setting from json/yml etc.
{
"metername" : "MeterForHttpLib"
, "instrumentname", "RequestCounts"
, keys : ["httpurl","httpstatus"]
 "aggregation":
    {
     Type:, "Histogram",
     Options: 
     {
       histogramboundaries....
      }
   }
}
When a meter/instrument matching the above is actually created in the program, it'll get its View config automatically from the config.

Thougts?

Reply:
I also think that the application owner may not have access to `requests_counter` object, if it was an instrument created inside another library. Only that library will have access to it. 
The application owner will have to use "metername + instrumentname" or similar to specify an instrument.

Reply:
Here's what I did for .net prototype.

```
MetricViewRegistry metricViewRegistry = new MetricViewRegistry();
metricViewRegistry.AddMetricView(new MetricView() { MeterName = "library1", InstrumentName = "testCounter", Aggregation = Aggregation.SUM, Keys = new List<string>() { "k1", "k2" } });
metricViewRegistry.AddMetricView(new MetricView() { MeterName = "library1", InstrumentName = "testCounter", Aggregation = Aggregation.SUM, Keys = new List<string>() { "k1" } });
metricViewRegistry.AddMetricView(new MetricView() { MeterName = "library1", InstrumentName = "testCounter", Aggregation = Aggregation.SUM, Keys = new List<string>() { "k2" } });
```

Reply:
This might be a possible improvement in the future. For the prototype, I believe it is okay to have just the basic functionality and iterate after.

Reply:
@cijothomas I like what you posted here. We should be imagining that views will be configured from a `.yaml` file, which many but not all users will want.